.cirrus.yml

```diff
@@ -19,7 +19,7 @@
 # Constants
 
 iox2_task_timeout_template: &IOX2_TASK_TIMEOUT
-  timeout_in: 30m  # This needs to be reasonable large in order not to run in a timeout in the docker image re-builds
+  timeout_in: 45m  # This needs to be reasonable large in order not to run in a timeout in the docker image re-builds
 
```

Comment:
It seems that FreeBSD 13.2 is no longer supported by CirrusCI

Reply:
Yes, we need to switch to FreeBSD 14.0